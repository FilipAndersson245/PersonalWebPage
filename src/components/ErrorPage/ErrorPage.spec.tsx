import { ErrorPage } from ".";
import renderer from "react-test-renderer";
import React from "react";
describe("styling", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<ErrorPage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
