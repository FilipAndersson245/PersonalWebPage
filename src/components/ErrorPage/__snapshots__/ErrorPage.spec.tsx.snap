// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styling should match snapshot 1`] = `
<div
  className="MuiGrid-container-1 MuiGrid-justify-xs-center-18"
  style={
    Object {
      "marginTop": "16",
    }
  }
>
  <div
    className="MuiGrid-item-2 MuiGrid-grid-sm-6-49"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <h1
      className="MuiTypography-root-98 MuiTypography-h1-110 MuiTypography-gutterBottom-125"
    >
      404 Page not found
    </h1>
    <h6
      className="MuiTypography-root-98 MuiTypography-h6-115"
    >
      The page could not be found this may be because the page dosent exist or something is broken on the developers side.
    </h6>
  </div>
</div>
`;
